crates/biome_js_analyze/src/services/aria.rs

```diff
@@ -53,17 +53,23 @@ impl AriaServices {
             if let AnyJsxAttribute::JsxAttribute(attr) = attribute {
                 let name = attr.name().ok()?.syntax().text_trimmed().to_string();
                 // handle an attribute without values e.g. `<img aria-hidden />`
-                let Some(initializer) = attr.initializer() else {
-                    defined_attributes
-                        .entry(name)
-                        .or_insert(vec!["true".to_string()]);
-                    continue;
+                let values = if let Some(initializer) = attr.initializer() {
+                    let initializer = initializer.value().ok()?;
+                    if let Some(static_value) = initializer.as_static_value() {
+                        // handle multiple values e.g. `<div class="wrapper document">`
+                        static_value
+                            .text()
+                            .split(' ')
```

Comment:
You can use this instead: https://doc.rust-lang.org/std/primitive.str.html#method.split_whitespace

Reply:
@ematipico 
Thanks your advice, I have incorporated the changes you suggested!
2dcbc5e